### DISCUSSION TITLE
After downloading MacOS Ventura 13.4  I can't get Transmission to work

### DISCUSSION THREAD

**Discussion by GeorgeJohn26** [transmission/transmission#5553] [Support]
After downloading Mac OS Ventura 13.4 Update I can't get Transmission to download videos . Can anyone kindly help. Thanks

**Comment by dantearmok** [comment]
Please be more specific.  Step by step...

**Comment by sfrimboy** [comment]
I've been having the exact same issue. My torrent files are uploading with Transmission, but I'm unable to download any content. When I download and double-click the torrent file, nothing happens. Is there a workaround for this?
